libs/shared/flagd-core/src/lib/feature-flag.ts

```diff
@@ -17,13 +17,13 @@ export class FeatureFlag {
   private readonly _state: string;
   private readonly _defaultVariant: string;
   private readonly _variants: Map<string, FlagValue>;
-  private readonly _targetingString: string;
+  private readonly _targeting: any;
```

Comment:
```suggestion
  private readonly _targeting: unknown;
```

This is safer and since you're not accessing props on it, it should be easy to use this way (might need to add an assertion or two).

Reply:
Done, changed to unkown